src/components/Icons/Twemoji/Camera.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+const SvgTwemoji1F4F7 = props => (
+  <svg viewBox="0 0 47.5 47.5" {...props}>
+    <defs>
+      <clipPath id="Twemoji_1f4f7_svg__a">
+        <path d="M0 38h38V0H0v38z" />
+      </clipPath>
+    </defs>
+    <g
+      clipPath="url(#Twemoji_1f4f7_svg__a)"
+      transform="matrix(1.25 0 0 -1.25 0 47.5)"
+    >
+      <path
+        d="M1 27s0 4 4 4h28s4 0 4-4V9s0-4-4-4H5S1 5 1 9v18z"
+        fill="#292f33"
+      />
+      <path
+        d="M12 18c0-5.523 4.477-10 10-10s10 4.477 10 10-4.477 10-10 10-10-4.477-10-10"
+        fill="#ccd6dd"
+      />
+      <path d="M30 18a8 8 0 00-8-8 8 8 0 100 16 8 8 0 008-8" fill="#292f33" />
+      <path
+        d="M27 18a5 5 0 00-5-5 5 5 0 00-5 5 5 5 0 005 5 5 5 0 005-5"
+        fill="#3b88c3"
+      />
+      <path d="M35 27.5a1.5 1.5 0 10-3 0 1.5 1.5 0 003 0" fill="#fff" />
+      <path
+        d="M13 27.5a1.5 1.5 0 00-1.5-1.5h-5a1.5 1.5 0 100 3h5a1.5 1.5 0 001.5-1.5"
+        fill="#f5f8fa"
+      />
+      <path d="M5 32s0 1 1 1h6s1 0 1-1v-1H5v1z" fill="#66757f" />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)